test/e2e/geojson/geojson2mapml.test.js

```diff
@@ -0,0 +1,139 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("GeoJSON API - geojson2mapml", () => {
+    let page;
+    let context;
+    test.beforeAll(async function() {
+      context = await chromium.launchPersistentContext('');
+      page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+      await page.goto("geojson2mapml.html");
+      //await page.waitForTimeout(10000);
+    });
+
+    test.afterAll(async function () {
+      await context.close();
+    });
+
+  test("Point Geometry (string json)", async () => {
+    const out = await page.$eval(
+      "body > mapml-viewer#output > layer-:nth-child(1)", 
+      (node) => node.outerHTML
+    );
+    const exp = await page.$eval(
+      "body > mapml-viewer#expected1 > layer-", 
+      (node) => node.outerHTML
+    );
+    expect(out).toEqual(exp);

```

Comment:
I like the simplicity of these tests.  Also, it is surprising, but cool, that expect(out).toEqual(exp) will do a deep-equals on the retrieved nodes.